Homework check

Lewkurs opened the pull request:
There are still some bugs with this code that I am stuck on - primarily in the DAOImpl classes.

Lewkurs committed:
Add DAOImpl classes and xml/xsd files for all classes - refactored Services and DAO classes

Lewkurs committed:
Add DAOImpl classes and xml/xsd files for all classes - refactored Services and DAO classes

Dnelson16 reviewed (commented):
1. Add JABX annotations to the fields and attributes of your model classes. Example: @XmlElement(name = "first_name") over your firstName property. first_name - this should match with your xml file
2. (create a plan and organize my trip(with hotels flights and content, based on my prefs and money) is not included that I can see.
3.  I would also recommend a menu class to help with the added business logic ^
4. Complete all the different parsers.
5.  Create connection pool class.
6. Change all System.out.prints with logger.
7. Looks like you deleted your abstractDAO but the abstract class that you had looks good.

Lewkurs committed:
JSON, Remote MYSQL Server, Refine Menu for project & Parsers

Errors Still Found In:
ReviewsDAOImpl - Lines 71 & 103
DestinationBookingsDAOImpl - Lines 57 & 81 & 102
HotelBookingsDAOImpl - Lines 49 - 51, 67-69,  83 & 84,
